src/main/com/example012/v1/view/LotteryGameView.java

```diff
@@ -0,0 +1,36 @@
+package main.com.example012.v1.view;
+
+import main.com.example012.v1.model.Lottery;
+import main.com.example012.v1.model.StringConstant;
+
+import java.util.Set;
+
+public class LotteryGameView {
+
+    private final StringBuilder results = new StringBuilder();
+
+    public void printResults(Set<Integer> winningNumbers, int numberOfTickets, Lottery lottery, int[] rightGuesses) {
```

Comment:
tady by asi bylo lepší to rozkouskovat, at jsou v tom prin results
jeste treba dalsi tri metody
printCOmpanyInfo
printWinningNumbers
printTIckets
a rovnou to v nich tisknout .... nez to ukladat do jednoho velkoho stringu .... ale ok